plugins/amd-gpu/README.md

```diff
@@ -63,3 +63,10 @@ interfaces with the hardware.
       PSP ~~~ kernel
       kernel ~~~ fwupdengine
 ```
+
+## Owners
+
+Any anonymous user can submit a pull request to modify this plugin, but the following people should
```

Comment:
It's not "really" anonymous - they're authenticated GitHub users.

How about just "anyone"?

Reply:
Makes sense -- fixed -- thanks.